cluster-provision/images/vm-image-builder/README.md

```diff
@@ -73,3 +73,12 @@ $ kubectl apply -f <VMI yaml file>
 $ kubectl wait --for=condition=AgentConnected vmi $VMI_NAME --timeout 5m
 $ virtctl console testvm1
 ```
+
+### Using podman
+CRI_BIN environment variable controls which container runtime to use
+```bash
+$ export CRI_BIN=podman
+$ ./build-containerdisk.sh example
+$ 
+$ ./publish-containerdisk.sh example quay.io/example/example-containerdisk:latest
+```
```

Comment:
we don't need this now right ?
as it is auto detected

you might want instead to add that the CRI_BIN can be overriden by exporting CRI_BIN

Reply:
Done

Reply:
we still have it 
https://github.com/kubevirt/kubevirtci/pull/936/files#diff-5911465af9b7f64fa499beab92df12415c644505539d24ba87eaeb0f19f92f2aR76-R84

we dont need the export it, nor podman specific explanation
just a general one but thats we already have